app/src/main/java/org/mozilla/focus/fragment/BrowserFragment.java

```diff
@@ -48,8 +48,8 @@
 import org.mozilla.focus.broadcastreceiver.DownloadBroadcastReceiver;
 import org.mozilla.focus.customtabs.CustomTabConfig;
 import org.mozilla.focus.locale.LocaleAwareAppCompatActivity;
-import org.mozilla.focus.menu.BrowserMenu;
-import org.mozilla.focus.menu.WebContextMenu;
+import org.mozilla.focus.menu.browser.BrowserMenu;
```

Comment:
I don't think I have a better answer here but I just want to raise some confusion I had: I found `browser` and `context` to be unintuitive at first. I was thinking, `appmenu` or `threedot` and `webview` or `webcontextmenu` would be more appropriate but now I'm not so sure (e.g. if we introduce another activity, do we want it in the browser subpackage?).